tools/cdk-integ-tools/lib/integ-helpers.ts

```diff
@@ -337,7 +337,11 @@ export const DEFAULT_SYNTH_OPTIONS = {
         },
       ],
     },
+    // Enable feature flags for all integ tests
```

Comment:
Don't we want to simply enable *all* feature flags? (i.e. iterate over the keys and add them all, same as we do in the CLI when a new project is created).

Reply:
Eventually yes. See PR description.

Reply:
Also, this is in `cdk-integ-tools` and the list of feature flags is in `cx-api`. It would not be possible to obtain the list here.

Reply:
Actually, I see that `cdk-integ-tools` already depends on `cx-api`. I'll update this accordingly in a subsequent PR.